lib/application/pages/home/collections/collections_list_view.dart

```diff
@@ -0,0 +1,235 @@
+import 'dart:ui' as ui;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:layoutr/common_layout.dart';
+import 'package:memo/application/constants/colors.dart' as colors;
```

Comment:
Already mentioned below about the reexport. If it's possible, it would be a great improvements in the import list here

Reply:
Already added my 2 cents on why re-exporting in dart is not a good overall decision IMHO.

When IDEs (vscode) and other lint decisions are improved, we may want to discuss this again, but as things are today, there is more to lose than to gain if following a re-export entry file